app/src/main/java/com/naman14/timberx/ui/activities/MainActivity.kt

```diff
@@ -69,8 +71,62 @@ class MainActivity : AppCompatActivity() {
         setupUI()
     }
 
-    private fun setupUI() {
+    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
```

Comment:
This is just re-organizing so that private methods are at the bottom

Reply:
@Zhuinden that felt out of scope here 😉 don't want to change too much at a time

Reply:
Plus, if I do https://github.com/naman14/TimberX/issues/12 and use KOIN, view models are declared like `val viewModel by viewModel<MainViewModel>` so that will resolve this anyways